charts/argo-cd/values.yaml

```diff
@@ -23,6 +23,14 @@ global:
   #   hostnames:
   #   - git.myhostname
 
+# Override APIVersions
+# If you want to template helm charts but cannot access k8s API server
+# you can set api versions here
+APIOverrides:
+  enabled: false
+  certmanager: cert-manager.io/v1
+  ingress: networking.k8s.io/v1beta1
+
```

Comment:
Why do we need an `enabled` flag here? It would be simpler to leave the defaults on `""` or `<nil>`/`~`. Otherwise the if/else if/else condition does not work properly anymore.

Reply:
yes might make sense, now you can set APIoverrides enabled and you then need to override them for both certmanager and ingress. I'll rebuild it based on the vars being emtpy